waitsensor/waitsensor.go

```diff
@@ -44,15 +44,22 @@ func init() {
 	})
 }
 
+type BoundingBoxConfig struct {
+	XMin float64 `json:"x_min"`
+	XMax float64 `json:"x_max"`
+	YMin float64 `json:"y_min"`
+	YMax float64 `json:"y_max"`
+}
+
 // Config contains names for necessary resources
 type Config struct {
-	DetectorName    string                 `json:"detector_name"`
-	ChosenLabels    map[string]float64     `json:"chosen_labels"`
-	CountThresholds map[string]int         `json:"count_thresholds"`
-	CountPeriod     float64                `json:"sampling_period_s"`
-	NSamples        int                    `json:"n_samples"`
-	ExtraFields     map[string]interface{} `json:"extra_fields"`
-	ValidRegions    map[string][][]float64 `json:"valid_regions"`
+	DetectorName    string                         `json:"detector_name"`
+	ChosenLabels    map[string]float64             `json:"chosen_labels"`
+	CountThresholds map[string]int                 `json:"count_thresholds"`
+	CountPeriod     float64                        `json:"sampling_period_s"`
+	NSamples        int                            `json:"n_samples"`
+	ExtraFields     map[string]interface{}         `json:"extra_fields"`
+	ValidRegions    map[string][]BoundingBoxConfig `json:"valid_regions"`
```

Comment:
You'll def want to manually test this on a local machine to see if it works as intended